geomesa-spark/geomesa-spark-jts/src/test/scala/org/locationtech/geomesa/spark/jts/util/RowGeoJSONTest.scala

```diff
@@ -0,0 +1,161 @@
+/***********************************************************************
+ * Copyright (c) 2013-2018 Commonwealth Computer Research, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, Version 2.0
+ * which accompanies this distribution and is available at
+ * http://www.opensource.org/licenses/apache2.0.php.
+ ***********************************************************************/
+
+package org.locationtech.geomesa.spark.jts.util
+
+import com.vividsolutions.jts.geom._
+import org.json.simple.JSONObject
+import org.json.simple.parser.JSONParser
+import org.junit.runner.RunWith
+import org.locationtech.geomesa.spark.jts._
+import org.specs2.mutable.Specification
+import org.specs2.runner.JUnitRunner
+
+@RunWith(classOf[JUnitRunner])
+class RowGeoJSONTest extends Specification with TestEnvironment {
+
+  "Row to GeoJSON converter" should {
+    sequential
```

Comment:
(Also picky, hey, I have to do something to feel good about myself) 
Does this really need to be sequential?